training/distributed_training/pytorch/data_parallel/roberta/roberta.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>Do you use PT 1.9 or PT 1.11? Please match the version number in #1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>